test/NftReward.t.sol

```diff
@@ -47,7 +57,8 @@ contract NftRewardTest is Test {
         // get mint request digest which should be signed
         bytes32 digest = nftReward.getMintRequestDigest(mintRequest);
 
-        assertEq(digest, 0x2c680706f2350ed5622f229af6736cd20626f7b9b4569b2fd5abb7e086886dc3);
+        // expected digest: 0x2c680706f2350ed5622f229af6736cd20626f7b9b4569b2fd5abb7e086886dc3
```

Comment:
This comment is ambiguous because it does not match the asserted hash, pls update or remove